haystack_experimental/super_components/converters/multi_file_converter.py

```diff
@@ -20,7 +19,7 @@
 from haystack.components.joiners import DocumentJoiner
 from haystack.components.routers import FileTypeRouter
 
-from haystack_experimental.core.super_component import SuperComponent
+from haystack_experimental.core.super_component import super_component
```

Comment:
Just a comment: Similar to how users can call from `haystack import component`, I'd say we should also enable them to call `haystack import super_component ` if possible. No need to enable `from haystack_experimental import super_component` here but that's how I imagine it.